### PR TITLE
Helper methods to create updated attestation.

### PR DESCRIPTION
**Pull request by ericsson49**

### CODE REVIEW

File: core/src/main/java/org/ethereum/beacon/core/operations/Attestation.java [hunk 1]
```diff
@@ -79,6 +81,24 @@ public BLSSignature getSignature() {
     return signature;
   }
 
+  public Attestation withAggregationBits(Bitlist aggregationBits, SpecConstants specConstants) {
+    return new Attestation(
+        ensureMaxSize(aggregationBits, specConstants), data, custodyBits, BLSSignature.ZERO);
+  }
+
+  public Attestation withData(AttestationData data) {
+    return new Attestation(aggregationBits, data, custodyBits, BLSSignature.ZERO);
```
Comment (mkalinin) [inline core/src/main/java/org/ethereum/beacon/core/operations/Attestation.java:92]: What if we add an explicit check like `assert signature == BLSSignagure.ZERO`?

Comment (ericsson49) [reply core/src/main/java/org/ethereum/beacon/core/operations/Attestation.java:92]: To forbid updating already signed attestations?

Comment (mkalinin) [reply core/src/main/java/org/ethereum/beacon/core/operations/Attestation.java:92]: Signed attestation looks like an invariant that we might want to preserve. WDYT?

Comment (ericsson49) [reply core/src/main/java/org/ethereum/beacon/core/operations/Attestation.java:92]: It makes sense. Updated the code.

**Review by mkalinin — APPROVED** [review]
LGTM! 👍

**Pull request merged**